apps/remote_control/lib/lexical/remote_control/build/error.ex

```diff
@@ -17,6 +37,70 @@ defmodule Lexical.RemoteControl.Build.Error do
     )
   end
 
+  defp normalize(%Result{} = result) do
+    result
+  end
+
+  defp format(%Result{} = result) do
+    %Result{result | message: format_message(result.message)}
+  end
+
+  defp format_message("undefined" <> _ = message) do
+    # All undefined messages explain the *same* thing inside the parentheses,
+    # like: `undefined function print/1 (expected Foo to define such a function or for it to be imported,
+    #        but none are available)`
+    # that makes no sense and just creates noise.
+    # So we can remove the things in parentheses
+
+    message
+    |> String.split(" (")
+    |> List.first()
+  end
+
+  defp format_message(message) when is_binary(message) do
+    maybe_format_unused(message)
+  end
+
+  defp maybe_format_unused(message) do
+    # Same reason as the `undefined` message above, we can remove the things in parentheses
+    case String.split(message, "is unused (", parts: 2) do
+      [prefix, _] ->
+        prefix <> "is unused"
+
+      _ ->
+        message
+    end
+  end
+
+  defp reject_zero_line(diagnostics) do
```

Comment:
obscure english: this would be the `reject_zeroth_line`

I cannot stress how odd of a convention this is.